index.js

```diff
@@ -1,84 +1,13 @@
-const Joi = require('joi');
-const cors = require('cors');
 const express = require('express');
-const { request } = require('express');
-const db = require("./user-votes");
-
-
+const votingsRoutes = require('./routes/votings/votings-api');
+const users = require('./routes/users/users-api');
 
 const app = express();
-app.use(cors({
-    origin: 'http://localhost:3000'
-  }));
-app.use(express.json());
-
-// const userVotes = [];
-
-//get all votes 
-app.get("/api/userVotes", async (req, res) => {
-    const allVotes = await db.getAllVotes();
-    res.send(allVotes);
-});
-
-//get part of votes by election id (foreign_key)
-app.get("/api/userVotes/:foreignKey", async (req, res) => {
-    const partOfVotes = await db.getVotes(req.params.foreignKey);
-    res.send(partOfVotes);
-});
-
-//insertion of new voting record
-app.post("/api/userVotes", async (req, res) => {
-    const {error} = validateCours(req.body)
-    if(error) return res.status(400).send(error.details[0].message)
-    const recordedVote = await db.createVote(req.body)
-    res.send(recordedVote)
-});
-
-app.put('/api/userVotes/:id', async (req, res) => {
-    const updatedVote = await db.updateVote(req.params.id, req.body)
-    if(!updatedVote) return res.status(404).send('The userVote with the given ID was not found.');
-
-    //validate 
-    //If invalid, return 400 - Bad request
-    // const {error} = validateCours(req.body)
-    // if(error) return res.status(400).send(error.details[0].message)
-       
-    console.log(updatedVote)
-    
-    // res.send(updatedVote)
-    //Return the updated userVote
 
-})
+// Mount route files
```

Comment:
I would remove this comment because it's obvious what the code is doing.

Reply:
Later 💯